framework/base/Application.php

```diff
@@ -246,6 +246,18 @@ public function preInit(&$config)
             $this->setTimeZone('UTC');
         }
 
+        if (isset($config['container'])) {
```

Comment:
It is done to make possible to set `ErrorHandler` in `container['definitions']` and have an effect.
Without this line, ErrorHandler will be registered after `ErrorHandler` instantiating in `registerErrorHandler()` method